CONTRIBUTING.md

```diff
@@ -125,130 +93,85 @@ Provide the following information:
 2. Follow all instructions in [PULL-REQUEST-TEMPLATE.md](.github/pull_request_template.md).
-3. Include appropriate test coverage. Testing is 100% automated. There is no such thing as a manual test.
-4. Follow the [Style Guides](#style-guides).
-5. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) 
+3. Follow the [Style Guides](#style-guides).
+4. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) 
 are passing.
 <details><summary>What if the status checks are failing?</summary>If a status check is failing, 
 and you believe that the failure is unrelated to your change, please leave a comment on the pull request 
 explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. 
 If we conclude that the failure was a false positive, then we will open an issue to track that problem 
 with our status check suite.</details>
 
-While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) 
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewers 
 may ask you to complete additional design work, tests, or other changes before your pull request 
-can be ultimately accepted.  Please refer to [Code Reviews].
+can be ultimately accepted.
 
 # Style Guides
 
-## Java Code Style Guide
-
-We use Google's Java coding conventions for the project. To reformat code, run: 
-
-```
-./gradlew spotlessApply
-```
-
-Doc style will be checked automatically during a build.
-
 ## Documentation Style Guide
-For documentation, we have [documentation guidelines and examples](DOC-STYLE-GUIDE.md). 
-These rules are not automatically enforced but are recommended to make the documentation consistent
- and enhance the user experience.
+Use the [documentation guidelines and examples](DOC-STYLE-GUIDE.md) when you suggest documentation changes.
+The guidelines are not automatically enforced but are recommended to make the documentation consistent and enhance the user experience.
 
-Also have a look at our [MKDocs Markdown guide](MKDOCS-MARKDOWN-GUIDE.md) if you're not famililar with 
-MarkDown syntax. We also have a number of extensions that are available in the EthSigner documentation described
-in this guide.
+If you are unfamiliar with Markdown syntax, see our [MKDocs Markdown guide](MKDOCS-MARKDOWN-GUIDE.md). Additionally, we have extensions that are available in the EthSigner documentation described in this guide.
 
-## Git Commit Messages & Pull Request Messages
-* Use the present tense ("Add feature" not "Added feature")
-* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
-* Provide a summary on the first line with more details on additional lines as needed
-* Reference issues and pull requests liberally
+## Git Commit Messages & Pull-Request Messages
```

Comment:
Having a ligne like the one in [EthSigner contribution file](https://github.com/PegaSysEng/ethsigner/blob/master/CONTRIBUTING.md#contribution-workflow) to point to Chris Beams' article may be better as we started to use this as our commit messages guidelines.
This paragraph was completely removed from EthSigner contribution file, so maybe it could be removed here too.

Reply:
I will add this link.